doc/docker/install.md

```diff
@@ -0,0 +1,44 @@
+## Why run Huginn with docker
+
+You can play with or deploy Huginn inside of [docker](http://www.docker.io/).
+
+Getting Huginn up and running using docker is quick and painless once you have docker installed. The docker container is suitable for production and evaluation. Huginn uses environmental variables for configuration, so rather than having a .env file, the Docker container expects variables to be passed into the launch command.
+
+## Running the Container
+
+### Quick start to check out Huginn
+
+#### OSX GUI using Kitematic
+
+1. Download and install [Kitematic](https://www.docker.com/docker-kitematic)
+* Start Kitematic and search for `cantino/huginn`
+* Click `create` and wait for the container to be downloaded and booted
+* Click on the link icon next to 'WEB PREVIEW'
+* Log in to your Huginn instance using the username `admin` and password `password`
+
+#### OSX/Windows/Linux using docker machine
+
+1. Download [docker machine](https://docs.docker.com/machine/#installation) for your OS
+* Follow the installation instructions untill you can successfully run `docker ps`
+* Get the the IP of the VM running docker by running `docker-machine ls`
+* Start your Huginn container using `docker run -it -p 5000:5000 cantino/huginn`
```

Comment:
Thanks, good catch, I think the default is now 3000, exposing 5000 does not seem to work for me anymore.